### DISCUSSION TITLE
To create a theme quickly and with fewer bugs, I decided to use the method:

### DISCUSSION THREAD

**Discussion by MerlinTheCodemancer** [prasanthrangan/hyprdots#1602] [Show and tell]
The tactic is to take the theme with the closest style to what you want/want, make a copy and change the name of both the folder and the files in the following directories "At least those were mine"

.config/hyde/themes
.themes/

*{Example theme name: Customization_Default}*

for the new theme style Copy custom and modify in: ```.themes/Customization_Padrao/index.theme``` # so that it can be recognized by the pc
Edit files in :``` .config/hyde/themes/Customization_Padrao``` to modify colors and styles to standardize themes in ".themes"
and to add images and/or gifs just add them to the wallpapers folder

**Comment by rubiin** [comment]
Here some themes I found https://github.com/xsghetti/HyprCrux/tree/main/src/themes
